### ISSUE TITLE
[FEA] The NVTabular dataloader + sample weights

### ISSUE THREAD

**Issue by ethem-kinginthenorth** [NVIDIA/NVTabular#667]
**Is your feature request related to a problem? Please describe.**
I wish I could use the NVTabular dataloader to do what TF does with `sample weights`. Here is the link: https://www.tensorflow.org/guide/keras/train_and_evaluate#sample_weights

**Describe the solution you'd like**
from TF: 'A "sample weights" array is an array of numbers that specify how much weight each sample in a batch should have in computing the total loss. It is commonly used in imbalanced classification problems (the idea being to give more weight to rarely-seen classes).'

If this feature is already implemented in, I would appreciate a simple example.

Thanks

**Comment by benfred** [comment]
This should be implemented with #850. @marcromeyn can you work with  @bschifferer on an example?

**Comment by marcromeyn** [comment]
We added a [map function](https://github.com/NVIDIA/NVTabular/blob/main/nvtabular/loader/tensorflow.py#L272) to the `KerasSequenceLoader`. This can be used to pass the sample-weight to the model, an example of this can be found in [this](https://github.com/NVIDIA/NVTabular/blob/main/tests/unit/test_tf_dataloader.py#L142) test. Let us know if you run into any issues!
